panel/chat/message.py

```diff
@@ -479,16 +479,28 @@ def _include_stylesheets_inplace(self, obj):
             for o in obj.objects:
                 self._include_stylesheets_inplace(o)
 
+    def _include_message_css_class_inplace(self, obj):
+        if hasattr(obj, "objects"):
+            for o in obj.objects:
+                self._include_message_css_class_inplace(o)
+        elif isinstance(obj, str):
+            obj = Markdown(obj)
```

Comment:
Still don't understand this, my point was that you're creating this `Markdown` object but it's never used, so why even create it instead of skipping?!

Reply:
But `obj` is created, then modified and then never used again.